libparsec/crates/types/src/id.rs

```diff
@@ -314,7 +325,7 @@ impl TryFrom<&str> for DeviceID {
     fn try_from(s: &str) -> Result<Self, Self::Error> {
         const ERR: &str = "Invalid DeviceID";
         let (raw_user_id, raw_device_name) = s.split_once('@').ok_or(ERR)?;
-        let user_id = raw_user_id.parse().map_err(|_| ERR)?;
+        let user_id = raw_user_id.parse().map_err(|_| dbg!(ERR))?;
```

Comment:
why dbg ?

Reply:
That's something I forget to remove :sweat:

Reply:
done